static/src/js/components/KeywordPicker/KeywordPicker.jsx

```diff
@@ -361,27 +363,37 @@ const KeywordPicker = ({
         </span>
       </div>
 
+      {
+        includeRecommendedKeywords && (
+          <KeywordRecommendations formData={formData} onChange={onChange} />
+        )
+      }
+
       {/* Renders the list of added keywords with a remove button */}
-      <div className="p-3 mb-2 keyword-picker__added-keywords">
-        {
-          Object.values(removeEmpty(formData)).map((item, index) => (
-            <li key={JSON.stringify(Object.values(item))}>
-              {Object.values(item).join(' > ')}
-
-              <Button
-                onClick={() => handleRemove(index)}
-                variant="link"
-              >
-                <i
-                  aria-label="Remove"
-                  className="fa fa-times-circle remove-button text-red ps-1"
-                  role="img"
-                />
-              </Button>
-            </li>
-          ))
-        }
-      </div>
+      {
+        !includeRecommendedKeywords && (
+          <div className="p-3 mb-2 keyword-picker__added-keywords">
+            {
+              Object.values(removeEmpty(formData)).map((item, index) => (
+                <li key={JSON.stringify(Object.values(item))}>
+                  {Object.values(item).join(' > ')}
+
+                  <Button
```

Comment:
The `Button` component has an `Icon` prop that should allow you to pass the icon, rather than having it in the children